.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ignore generated folders
```

Comment:
Should we keep this file consistent between projects? Or, at least, add an ignore for `package-lock.json` just in case.